src/main/java/com/textmessenger/controller/UserController.java

```diff
@@ -13,8 +14,12 @@
 import com.textmessenger.service.EmailService;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'com.textmessenger.service.EmailService'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=236&issues=AWYK8i_OvOWp7Yd6HYt4&open=AWYK8i_OvOWp7Yd6HYt4)

[ISSUE_KEY]: AWYK8i_OvOWp7Yd6HYt4